README.md

```diff
@@ -1,6 +1,6 @@
 # babel-plugin-react-transform
 
-This Babel plugin wraps all React components into arbitrary transforms written by the community.  
```

Comment:
Whitespace is significant in Markdown. I put two spaces on purpose to force it to break line.